Test/MemorySafetryTests.cs

```diff
@@ -25,7 +25,9 @@ public class MemorySafetryTests
 
         [Test]
         [Parallelizable(ParallelScope.Children)]
-        public void DisposeDuringCapture([Range(1, 10)] int _)
+        // pcap_compile() in 1.8.0 and later is newly thread-safe
```

Comment:
what are the side effects here of not calling this routine? Are we not testing for behavior that we know will fail on these older libpcap revs? It's a little unclear what the intent is here. Also, filename should probably be MemorySafetyTests.cs (type in safety).

Reply:
I am simply disabling this test on old Libpcap until we fix it, since it's causing false positives for all currently open PRs, 

The topic would be addressed in #311

Reply:
@kayoub5 mind adding a comment that this is what you are doing? like:

//pcap_compile() in 1.8.0 and later is newly thread-safe
// disable dispose during capture tests in earlier versions to avoid crashes due to known lack of thread-safety

Reply:
I moved those changes to #312, could we have that one merged first?